catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -190,74 +193,12 @@ const SearchForm = ({
                 [spacing({ s: 1 }, { margin: ['top'] })]: true,
               })}
             >
-              <fieldset
-                className={classNames({
-                  relative: true,
-                  [spacing({ s: 2 }, { margin: ['right'] })]: true,
-                })}
-                style={{
-                  left: '1px',
-                }}
-              >
-                <legend
-                  className={classNames({
-                    'float-l': true,
-                    [font({ s: 'HNL4' })]: true,
-                  })}
-                  style={{ marginTop: '3px' }}
-                >
-                  Filter by:
-                </legend>
-                <SearchTag
-                  name={'workType'}
-                  label="Images"
-                  value="k,q"
-                  checked={
-                    workType.indexOf('k') !== -1 && workType.indexOf('q') !== -1
-                  }
-                  onChange={event => {
-                    const input = event.currentTarget;
-                    const newWorkType = input.checked
-                      ? [...workType, 'k', 'q']
-                      : workType.filter(val => val !== 'k' && val !== 'q');
-                    setWorkType(newWorkType);
-                  }}
-                />
-                <SearchTag
-                  name={'workType'}
-                  label="Books"
-                  value="a"
-                  checked={workType.indexOf('a') !== -1}
-                  onChange={event => {
-                    const input = event.currentTarget;
-                    const newWorkType = input.checked
-                      ? [...workType, 'a']
-                      : workType.filter(val => val !== 'a');
-                    setWorkType(newWorkType);
-                  }}
-                />
-                <SearchTag
-                  name={'items.locations.locationType'}
-                  label="Online"
-                  value="iiif-image"
-                  checked={
-                    itemsLocationsLocationType.indexOf('iiif-image') !== -1
-                  }
-                  onChange={event => {
-                    const input = event.currentTarget;
-                    if (input.checked) {
-                      setItemsLocationsLocationType(['iiif-image']);
-                    } else {
-                      setItemsLocationsLocationType([]);
-                    }
-                  }}
-                />
-              </fieldset>
               <p
                 className={classNames({
                   [font({ s: 'HNL4' })]: true,
                   'no-margin': true,
                   relative: true,
+                  [spacing({ s: 2 }, { margin: ['right'] })]: true,
```

Comment:
Does this win over `no-margin` above?

Reply:
Correcto - maybe not the way to express it, but it works.

Reply:
Strange – I'm sure I've encountered a situation where `no-margin` wins.

Reply:
![screenshot 2019-02-26 at 11 14 54](https://user-images.githubusercontent.com/31692/53409030-e11bba80-39b7-11e9-8ed3-70d95d42cd66.png)

Correct, now fixed.